aixplain/modules/pipeline_response.py

```diff
@@ -0,0 +1,54 @@
+from dataclasses import dataclass
+from typing import Any, Optional, Dict, List, Union, Text
+from aixplain.enums import Status 
+
+@dataclass
+class PipelineResponse:
+    
+    def __init__(
```

Comment:
The use of dataclass here is not so correct. Dataclasses mostly designed to auto populate its attributes from the data fields defined. And also some other meta information also could be attached by mixing the use of both `InitVar` and the `__post_init__` function.

Reply:
Yes or no depends on your decision. In this version, you're almost using a regular class, no advantage in using dataclass here.